copy/copy.go

```diff
@@ -191,9 +191,16 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		return err
 	}
 
+	var manifestMIMEType []string
+	if options.ManifestType != "" {
+		manifestMIMEType = append(manifestMIMEType, options.ManifestType)
+	} else {
+		manifestMIMEType = dest.SupportedManifestMIMETypes()
+	}
```

Comment:
This does work, but it requires a not-entirely-obvious inspection of `determineManifestConversion` to verify that it does.  Could you instead add a `forceManifestMIMEType` option to `determineManifestConversion`, do this in there, and perhaps add some tests to ensure that we always return exactly that one MIME type?

Reply:
fixed